index.bs

```diff
@@ -2220,7 +2226,13 @@ Base64-encoded string.
 
       browsingContext.CaptureScreenshotParameters = {
         context: browsingContext.BrowsingContext,
-        ? clip: browsingContext.ClipRectangle
+        ? format: "png" / "webp" / browsingContext.JpegFormat .default "png",
+        ? clip: browsingContext.ClipRectangle,
+      }
+
+      browsingContext.JpegFormat = {
+         type: "jpeg",
+         ? quality: 0.0..1.0,
```

Comment:
Should quality have a default?

Reply:
Quality has [a default set defined by the browser.](https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toDataURL#:~:text=is%20not%20supported.-,encoderOptions,-Optional)